presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/StateCompiler.java

```diff
@@ -244,7 +243,7 @@ else if (fields.size() == 1) {
         }
         else if (fields.size() > 1) {
             Variable row = scope.declareVariable(Block.class, "row");
-            deserializerBody.append(row.set(block.invoke("getObject", Object.class, index).cast(Block.class)));
+            deserializerBody.append(row.set(block.invoke("getObject", Object.class, index)));
```

Comment:
What about? 

```
deserializerBody.append(row.set(block.invoke("getBlock", Block.class, index)))
```

Sorry I forgot to change `getObject` to `getBlock`, thus the method name still not found.